Specs/DateTime/Japanese/DateTimeExtractor.json

```diff
@@ -458,14 +458,7 @@
   {
     "Input": "日曜日のうちに戻ります。",
     "NotSupportedByDesign": "javascript,python,java",
-    "Results": [
-      {
-        "Text": "日曜日のうちに",
-        "Type": "datetime",
-        "Start": 0,
-        "Length": 7
-      }
-    ]
+    "Results": []
```

Comment:
Extracted in date, correct?

Reply:
yes, the case is in the model where it resolves to date.